src/numeric-values.js

```diff
@@ -405,4 +423,422 @@ export function multiplyTypes(type1, type2) {
     finalType[baseType] += type2[baseType];
   }
   return finalType;
+}
+
+class CSSFunction {
+  name;
+  values;
+  constructor(name, values) {
+    this.name = name;
+    this.values = values;
+  }
+}
+
+class CSSSimpleBlock {
+  value;
+  associatedToken;
+  constructor(value, associatedToken) {
+    this.value = value;
+    this.associatedToken = associatedToken;
+  }
+}
+
+/**
+ * Normalize into a token stream
+ * https://www.w3.org/TR/css-syntax-3/#normalize-into-a-token-stream
+ */
+function normalizeIntoTokenStream(input) {
+  // If input is a list of CSS tokens, return input.
+  // If input is a list of CSS component values, return input.
+  if (Array.isArray(input)) {
+    return input;
+  }
+  // If input is a string, then filter code points from input, tokenize the result, and return the final result.
+  if (typeof input === 'string') {
+    return tokenizeString(input);
+  }
+  // Assert: Only the preceding types should be passed as input.
+  throw new TypeError(`Invalid input type git${typeof input}`)
```

Comment:
I think the text "git" prepended before the type may have been unintentional?

Reply:
Oh.. that's embarrassing 😅
I'm having an issue where the focus is delayed when switching between the IDE and the terminal 😞

Fixed in https://github.com/flackr/scroll-timeline/compare/8e7cf381430bd1d2c7511debec7d954ebaccaf66..4ed64d0898c93a488381f6ad35c42033b39698a8